src/async-wrap.cc

```diff
@@ -121,18 +122,35 @@ static void SetupHooks(const FunctionCallbackInfo<Value>& args) {
 
   if (env->async_hooks()->callbacks_enabled())
```

Comment:
I'm missing context. Users have to explicitly run `async_wrap.enable()` before callbacks will be called. This case is to handle:
```js
const async_wrap = require('async_wrap');
async_wrap.setupHooks({ ...callbacks... });
async_wrap.enable();
async_wrap.setupHooks({ ...different callbacks... });
```
To prevent callbacks from being pulled out from underneath the user while operations may be in flight.